pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -459,12 +463,19 @@ RUN chmod +x /usr/local/bin/start_with_right_hostname.sh
 WORKDIR /root
 
 RUN pip install --no-cache-dir -U \
-    smdebug==${SMDEBUG_VERSION} \
+    # smdebug==${SMDEBUG_VERSION} \
     smclarify \
     "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
     sagemaker-pytorch-training
 
+# Install smdebug from souce
+RUN cd /tmp \
+  && git clone -b ${SMDEBUG_VERSION} https://github.com/awslabs/sagemaker-debugger \
```

Comment:
why install from source? Why not from pip directly?

Reply:
There's no `smdebug 1.0.16` pypi wheels. That's a decision made by the `smdebug` team.

Reply:
@ydaiming can you add a comment above where you comment out smdebug, saying you are installing from source bc the version is not available from pypi?